Feat(apple): Add missing information regarding SwiftUI (#6788)
getsentry/sentry-docs: src/platforms/apple/common/performance/instrumentation/automatic-instrumentation.mdx

## User Interaction Tracing

> User interaction tracing, once enabled, captures transactions for clicks (this do not work with SwiftUI). Since 8.0.0, this feature has been enabled by default. To disable it:

Reviewer: `m`: Maybe the docs team has a better idea for the proper wording.
```suggestion
User interaction tracing, once enabled, captures transactions for clicks. This feature is unavailable for SwiftUI. Since 8.0.0, this feature has been enabled by default. To disable it:
```
